build.sbt

```diff
@@ -3,26 +3,22 @@ ThisBuild / githubOrganization := "47degrees"
 ThisBuild / scalaVersion := V.scala
 ThisBuild / crossScalaVersions := Seq(V.scala212, V.scala)
```

Comment:
Why does this project needs to cross-compile?

Reply:
Originally was part of the sbt-exercise plugin. Now, it can be removed 👍